src/common/playlists/DynamicBanner.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import thumbPlay from 'images/thumb-play.png';
+import { Link } from 'react-router-dom';
+import { MdArrowRightAlt } from 'react-icons/md';
+
+const DynamicBanner = ({ randomPlay }) => {
+  const [coverImage, setCoverImage] = useState(null);
+
+  useEffect(() => {
+    if (randomPlay && randomPlay.cover) {
+      setCoverImage(randomPlay.cover);
+    } else {
+      import(`plays/${randomPlay.slug}/cover.png`)
+        .then((Cover) => {
+          setCoverImage(Cover.default);
+        })
+        .catch((err) => {
+          setCoverImage(thumbPlay);
+
+          return {
+            success: false,
+            error: err,
+            message: `Cover image not found for the play ${randomPlay.name}. Setting the default cover image...`
+          };
+        });
+    }
+  }, [randomPlay]);
+
+  return (
+    <Fragment>
+      <div
+        className="dynamic-banner-container"
+        style={{
+          background: `linear-gradient(rgba(0,0,0,0.5), #020808),url(${coverImage} ) center no-repeat`,
+          minHeight: '70vh'
```

Comment:
The banner height is too large and we end up seeing only one row of play. Reduce it to 50vh

![image](https://user-images.githubusercontent.com/3633137/213213881-2587b499-80dc-4d62-8288-43b7254233cc.png)

Also why the inline styles?